doc/input-output-reference/src/overview/group-surface-construction-elements.tex

```diff
@@ -4158,6 +4160,14 @@ \subsubsection{Outputs}\label{outputs-36-1}
 
 This output is the temperature within the surface at the location requested by the user.
 
+\paragraph{CondFD Internal Heat Source Power After Layer}
+
+This output is the heat power added after material layer N from the \hyperref[constructioninternalsource]{ConstructionProperty:InternalHeatSource} object. Only valid for the CondFD solution algorithm.
+
+\paragraph{CondFD Internal Heat Source Energy After Layer}
```

Comment:
The two new "CondFD EMS Heat Source ..." outputs should be here also. Again, in the next PR.